docs/core/core-endpoints.md

```diff
@@ -3,37 +3,57 @@ id: conflux_rpcs
 title: Network RPC Endpoints
 sidebar_position: 4
 keywords:
-  - endpoints
+  - network endpoints
+  - Conflux core space
 displayed_sidebar: coreSidebar
 ---
 
 Public available Conflux Core Space network RPC endpoints
 
-## 1. Confura
+:::tip
+
+Public RPC endpoints for **Conflux eSpace** is provided [here](../espace/network-endpoints.md).
 
-Confura is an Ethereum Infura equivalent public JSON-RPC service on Conflux network, which is developed and maintained by Conflux foundation.
+:::
 
-Our public RPC services are located in different regions globally. By utilizing geographic DNS routing, developers can request the regional RPC services which are physically closest to them.
+## 1. Confura
+
+Confura is an Ethereum Infura equivalent public JSON-RPC service on Conflux network, which is developed and maintained by Conflux foundation and is available for free use.
 
 ### Public Endpoint
 
-#### Hong Kong
+Confura provides several available RPC endpoints for the mainnet and testnet. We have sorted them by selected priority and provided brief descriptions. If you are unsure which one to choose, please select an RPC endpoint near the top of the list.
 
-| Network | Chain ID | Explorer | RPC Endpoint | 
-| -------- | -------- | --------| -------- |
-| Mainnet | 1029 | https://confluxscan.net | *HTTP*: https://main.confluxrpc.com<br/>*Websocket*: wss://main.confluxrpc.com/ws |
-| Testnet | 1 | https://testnet.confluxscan.net | *HTTP*: https://test.confluxrpc.com<br/>*Websocket*: wss://test.confluxrpc.com/ws |
+#### Mainnet
 
-#### US East
+Chain ID for Conflux Core Space Mainnet is `1029(0x405)`. The corresponding blockchain explorer URL is https://confluxscan.io.
 
-| Network | Chain ID | Explorer | RPC Endpoint | 
-| -------- | -------- | --------| -------- |
-| Mainnet | 1029 | https://confluxscan.io | *HTTP*: https://main.confluxrpc.org<br/>*Websocket*: wss://main.confluxrpc.org/ws |
-| Testnet | 1 | https://testnet.confluxscan.io | *HTTP*: https://test.confluxrpc.org<br/>*Websocket*: wss://test.confluxrpc.org/ws |
+| RPC Endpoint | Type | Notes |
+| -------- | -------- | --- |
+| https://main.confluxrpc.com | HTTP | |
```

Comment:
Do you guys @darwintree  @confluxdevops  think we need to add a note to indicate this endpoint uses smart DNS routing?

Reply:
No need to introduce the system architecture.